### PR TITLE
Delete old give page

### PR DESCRIPTION
**Pull request by CryptoGraffe**
## Motivation and Context

Old give page is no longer linked or used, we can finally jettison it into the sun. This changes /givebeta to /give , deletes some recoil stuff, and changes the cypress test to use the new give page. 

## Description

Delete the page and some of the dependencies that are no longer used. 

## Test and Deployment Plan

Make sure nav and allocating work ok.

### CODE REVIEW

File: cypress/e2e/give.cy.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      circleId = q.circles[0].id;
+    });
+  });
+  it('can login and visit give', () => {
+    cy.visit(`/circles/${circleId}/give`);
+    cy.login();
+    cy.url({ timeout: 120000 }).should('include', '/give');
+    cy.contains('thank your teammates').click();
+  });
+  // it('can select teammates', () => {
```
Comment (crabsinger) [inline cypress/e2e/give.cy.ts:25]: should we make this try the "+ Collaborator" thing when you hover on a row?

File: src/pages/GivePage/queries.ts [hunk 2]
```diff
@@ -122,6 +122,141 @@ export type CircleAllocationText = Awaited<
...
+            user: {
+              circle_id: { _eq: selectedCircleId },
+              address: { _ilike: address },
+            },
+          },
+        },
+        {
+          team_mate_id: true,
+          teammate: {
+            // id: true,
```
Comment (crabsinger) [inline src/pages/GivePage/queries.ts:221]: why commented? remove if not needed?

**Review by crabsinger — DISMISSED** [review]
@CryptoGraffe two small nits - otherwise looks good

File: cypress/e2e/give.cy.ts (hunk 1 above)
Comment (CryptoGraffe) [reply cypress/e2e/give.cy.ts]: did it

File: src/pages/GivePage/queries.ts (hunk 2 above)
Comment (CryptoGraffe) [reply src/pages/GivePage/queries.ts]: this whole query can be deleted , nice catch

**Review by levity — APPROVED** [review]

**Pull request merged**